src/validation.cpp

```diff
@@ -1956,6 +1956,29 @@ const CBlockIndex* Chainstate::SnapshotBase() const
     return m_cached_snapshot_base;
 }
 
+const CBlockIndex* Chainstate::TargetBlock() const
+{
+    if (!m_target_blockhash) return nullptr;
+    if (!m_cached_target_block) m_cached_target_block = Assert(m_chainman.m_blockman.LookupBlockIndex(*m_target_blockhash));
+    return m_cached_target_block;
+}
+
+void Chainstate::SetTargetBlock(CBlockIndex* block) EXCLUSIVE_LOCKS_REQUIRED(::cs_main)
```

Comment:
0cb19b78cc297ceca591ee8be6f63b4ea7b494cc: Please don't add lock annotations to the cpp file. They won't be visible anyway. You'll have to add them to the .h file.

Reply:
re: https://github.com/bitcoin/bitcoin/pull/30214#discussion_r2570730621

> Please don't add lock annotations to the cpp file

Thanks, removed. (These were in the header file too and were probably just copied and pasted.)